velox/docs/functions/delta/functions.rst

```diff
@@ -0,0 +1,8 @@
+*****************
+Delta Lake Functions
+*****************
+
+Here is a list of all scalar Delta Lake functions available in Velox.
+Function names link to function description.
+
+(We don't have Delta Lake functions added so this documentation is empty. We will add them shortly.)
```

Comment:
Could you add the function link like https://github.com/facebookincubator/velox/pull/13207/files#diff-becf36f6e870124614abb95932682195b98698d0ef9e2dfcaf4e31dbc643dd21R11, so we can make sure the link work well.

Reply:
done.